apiserver/resources.go

```diff
@@ -213,7 +218,8 @@ func extractUploadRequest(req *http.Request) (api.UploadRequest, error) {
 		return ur, errors.Annotate(err, "invalid fingerprint")
 	}
 
-	filename, err := extractFilename(req)
+	// Extract out to extract value or filename etc.
+	resourceValue, err := extractFilename(req)
```

Comment:
This is the filename isn't it? I'm not sure about the rename to resourceValue.
Only file type resources will have the filename attribute in the req header set.
Or is the intent to use resourceValue to hold the docker path?